custom/yeksi_naa_reports/sqldata.py

```diff
@@ -190,49 +190,55 @@ class AvailabilityData(VisiteDeLOperateurDataSource):
 
     def calculate_total_row(self, rows):
         total_row = ['Availability (%)']
+        total_nominator = 0
+        total_denominator = 0
         if self.loc_id == 'pps_id':
             data = {}
             for i in range(len(self.months)):
                 data[i] = {
-                    'pps_is_outstock': 0,
-                    'pps_count': 0
+                    'pps_is_available': sum(
+                        [pps_data[i + 1] for pps_data in rows if pps_data[i + 1] != 'no data entered']
```

Comment:
Does this work if you omit the outermost `[]` ? That would create a generator expression and avoid creating an extra list